Source/WebCore/Modules/modern-media-controls/controls/time-control.js

```diff
@@ -42,7 +42,7 @@ class TimeControl extends LayoutItem
         this._shouldShowDurationTimeLabel = this.layoutTraits.supportsDurationTimeLabel();
 
         this.elapsedTimeLabel = new TimeLabel(TimeLabel.Type.Elapsed);
-        this.scrubber = new Slider(this.layoutDelegate, "scrubber");
+        this.scrubber = new (this.layoutTraits.scrubberSliderClass())(this.layoutDelegate, "scrubber");
```

Comment:
if this needs to be overridden, it could also easily be done by `this.timeControl.scrubber = ...`

i'd personally prefer that because this would make it harder to find places where `Slider` is used (i.e. i can't search for `new Slider` anymore :confused:)

Reply:
That's absolutely right. No work is needed here.